### DISCUSSION TITLE
disable spacebar to toggle button

### DISCUSSION THREAD

**Discussion by IttzyTT** [tailwindlabs/headlessui#1026] [Help]
Hey folks, is it possible to disable the buttons to be triggered by spacebar? We have the issue where we have a input and when we write and use space in text it closes the dropdown.

**Comment by ernes-to** [comment]
@IttzyTT If you could share a Codesandbox with reproduction steps I'm sure it'll be easier for others to help. In general though, the space bar triggering the `onClick` on a native `button` element is a default behavior that's important for accessibility reasons. If you need to disable that (highly discouraged practice) or handle it differently, then you could attempt to prevent the default behavior of the button or target the keycode for spacebar to make it do what you want.

**Comment by stathis1998** [comment]
I asked the guy who is working on tailwind and got this response just now.
"Hey! No that's not possible right now, and probably won't be anytime soon since we are following an accessibility spec. We are working on a combobox which essentially is an input + listbox, as we speak." - Robin

**<\/issue>**

**Comment by graigdev** [comment]
The same problem until now

**Reply by startupmasters** [reply]
Did you solve it somehow?
